functions/Backup-DbaDbCertificate.ps1

```diff
@@ -152,8 +152,8 @@ function Backup-DbaDbCertificate {
                 $actualPath = Get-SqlDefaultPaths -SqlInstance $server -filetype Data
             }
 
-            $actualPath = "$actualPath".TrimEnd('\')
-            $fullCertName = "$actualPath\$certName$Suffix"
+            $actualPath = "$actualPath".TrimEnd('\').TrimEnd('/')
```

Comment:
I think `.TrimEnd('/\')`should also work:
https://docs.microsoft.com/de-de/dotnet/api/system.string.trimend
But not sure if back to PS3.

Reply:
I tried it then a part failed, which may have been unrelated, so then I went back to the regular way